helm/templates/coder.yaml

```diff
@@ -25,6 +25,10 @@ spec:
     metadata:
       labels:
         {{- include "coder.labels" . | nindent 8 }}
+      annotations:
+        {{- with .Values.coder.podAnnotations}}
+        {{- toYaml . | nindent 8  }}
+        {{- end}}
```

Comment:
Instead of using the with syntax here, it's more concise to just use the single toYaml call with the full value name.